### PR TITLE
Bug fix

### PR DESCRIPTION
**Pull request by chvogl**
Bug fix for the atomic.base.py and for the atomic.plugins.py. We found a few syntax errors. 
From @mklauser this is WP, however we could merge it to atomic/ restructure.

### CODE REVIEW

**Comment by wkerzendorf** [comment]
@chvogl congratulations on your first PR! I'll have some comments inline for you.

File: tardis/atomic/base.py [hunk 1]
```diff
@@ -9,9 +9,9 @@ class AtomicData(object):
     def from_hdf(cls, fname):
         return cls(pd.HDFStore(fname, mode='r'))
 
-    def __init__(self, hdf_store, data_types):
+    def __init__(self, hdf_store, data_types=self.all_data_types):
```
Comment (wkerzendorf) [inline tardis/atomic/base.py]: this won't quite work. you can't access `self` in the initializer. the `data_types` should also be given by the user. so for now, I suggest leaving it as is.

File: tardis/atomic/base.py [hunk 2]
```diff
@@ -9,9 +9,9 @@ class AtomicData(object):
...
         self.data_types = [data_type()
```
Comment (wkerzendorf) [inline tardis/atomic/base.py]: I see the confusion. hmm - yes that was definitely not right what I wrote. I think we need to think a little more how we want this object to be initialized. has @mklauser given you an HDF5 file?

File: tardis/atomic/base.py [hunk 3]
```diff
@@ -19,7 +19,7 @@ def __init__(self, hdf_store, data_types):
 
     def __getattr__(self, item):
         if item in self.name2data_type:
-            return self.name2date_type[item].data
+            return self.name2date_type[item].value
```
Comment (wkerzendorf) [inline tardis/atomic/base.py]: very good catch! I missed that.

File: tardis/atomic/base.py (hunk 2 above)
Comment (chvogl) [inline tardis/atomic/base.py]: @mklauser hasn't given me an HDF5 file yet, but he wants to do so today.

**Comment by wkerzendorf** [comment]
@chvogl is this still relevant?

**Comment by chvogl** [comment]
@wkerzendorf
I'm sorry for taking so long to reply, but I just came back from my holidays.
To answer your question:
The last two items seem like they are still relevant, since there haven't been any changes in the atomic/restructure since then. However I'm not sure what the current plan for the atomic restructure is.

**Comment by wkerzendorf** [comment]
@chvogl is this still relevant. Can we close?

**Comment by chvogl** [comment]
To be honest, I have no idea what the current status regarding the atomic/restructure is and if we plan
on using any of it in the future.
I would assume that this PR has become irrelevant and that it can be closed but i cannot tell
you with absolute certainty.

**Comment by wkerzendorf** [comment]
@chvogl I will close it - we can reopen if necessary.

**Pull request closed without merging**